pages/db_views.py

```diff
@@ -0,0 +1,40 @@
+"""
+Views processing db based templates.
+This view are outside of MTV concept.
+Responsible only for rendering given context data with db preserved text template.
+"""
+import typing
+
+from pages import models
+
+
+class Fields:
+    # Fields stored in DB. See class `pages.models.PageTemplate`
+    STORED = ['name', 'h1', 'keywords', 'description', 'title', 'seo_text']
+
+    def __init__(self, page_view: 'Page'):
+        self.page_view = page_view
+
+    def __getattr__(self, item):
+        if item in self.STORED:
+            return self.page_view.render(item)
+
+
+class Page:
```

Comment:
```suggestion
class RenderedPage:
```

Reply:
@artemiy312 , let's look at the `shopelectro.views.catalog.CategoryPage` class for example.
Here we have pretty the same situation: view uses some template and given context to render response body. But we don't call it `RenderedCategoryPage`.

So, let's leave the `Page` name